models/iterable.yml

```
@@ -330,7 +332,7 @@ models:
       - name: user_updated_at
         description: Last update timestamp
       - name: list_id
-        description: The list associated with the user
+        description: The list associated with the user. Uses coalesced values prioritizing list_id from stg_iterable__list_user over the unnested email_list_ids from user_history for comprehensive list coverage.
```

```suggestion
        description: The list associated with the user. Uses coalesced values prioritizing `list_id` from `LIST_USER` over the unnested `email_list_ids` from `USER_HISTORY` for comprehensive list coverage.
```
Just to make it clearer where these are coming from